### PR TITLE
Update es_ES app translation for 3.9.0

### LINKED ISSUE #2693: Update es_ES app translation for 3.9.0
Hi @ignotus666,
We are getting ready for the 3.9.0 release. No further changes to translatable strings are currently planned for this release.

We would be happy if you updated the Jamulus software translations for **es_ES** by **2022-07-22**.

Please note the new location of the translation files (.ts): They are now in src/translation. (Refactoring: move src/res/translation to src/translation #2576)

Please
- Update your fork from `jamulussoftware/jamulus` `master` and create a working branch
- Update translations using Qt Linguist in your fork,
- Commit and push your changes and reference this Issue,
- Open a Pull Request before 2022-07-22

Suggested commit and PR message:
```
Update es_ES app translation for 3.9.0

Fixes #2693
```

Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/master/docs/TRANSLATING.md).

Thanks for contributing to Jamulus!

### PR DESCRIPTION
**Pull request by ignotus666**
**Short description of changes**

Update es_ES app translation for 3.9.0

CHANGELOG: Updated Spanish translation of app for 3.9.0

**Context: Fixes an issue?**

Fixes #2693

**Does this change need documentation? What needs to be documented and how?**

No

**Status of this Pull Request**

Ready to be merged

**What is missing until this pull request can be merged?**

## Checklist

-  [ ] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [ ] I tested my code and it does what I want
-  [ ] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
-  [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [ ] I've filled all the content above

### CODE REVIEW

File: src/translation/translation_es_ES.ts [hunk 1]
```diff
@@ -3585,113 +3585,114 @@ Hemos silenciado tu canal y activado &apos;Silenciarme Yo&apos;. Por favor resue
...
         <location filename="../serverdlg.cpp" line="67"/>
         <source>For any value except &apos;%1&apos;, this server registers with a directory so that a %2 user can select this server in the client connect dialog server list when they choose that directory.</source>
-        <translation type="unfinished"></translation>
+        <translation>Para cualquier valor excepto &apos;%1&apos;, este servidor se registra en un directorio para que un usuario de %2 pueda seleccionar este servidor de la lista de servidores en la ventana de conexión del cliente cuando escogen ese directorio.</translation>
     </message>
     <message>
         <location filename="../serverdlg.cpp" line="73"/>
         <source>The registration of the server is renewed periodically to make sure that all servers in the connect dialog server list are actually available.</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>El registro del servidor se renueva periódicamente para asegurarse de que todos los servidores en la lista de servidores en la ventana de conexión están realmente disponibles.</translatorcomment>
```
Comment (ann0see) [inline src/translation/translation_es_ES.ts:3618]: Why did you translate a comment and left the translation empty? Is this an error?

File: src/translation/translation_es_ES.ts [hunk 2]
```diff
@@ -3585,113 +3585,114 @@ Hemos silenciado tu canal y activado &apos;Silenciarme Yo&apos;. Por favor resue
...
         <location filename="../serverdlg.cpp" line="212"/>
         <source>Server List Filename text box (read-only)</source>
-        <translation type="unfinished"></translation>
+        <translation>Caja texto Nombre Archivo Lista de Servidores (solo lectura)</translation>
     </message>
     <message>
         <location filename="../serverdlg.cpp" line="214"/>
         <source>The current value of server list persistence file name. The user %1 is running as needs to be able to create the file name specified although it may already exist (it will get overwritten on save). Click the button to open the dialog that allows the server list persistence file name to be set.</source>
-        <translation type="unfinished"></translation>
+        <translation>Valor actual del nombre del archivo de persistencia para la lista de servidores. El usuario ejecutando %1 debe poder crear el nombre de archivo especificado aunque ya exista (se sobreescribirá al guardar).Pulsa el botón para abrir la ventana que permite establecer el nombre del archivo de persistencia para la lista de servidores.</translation>
```
Comment (ann0see) [inline src/translation/translation_es_ES.ts:3680]: Probably a space is missing before "Pulsa". But idk about Spanish punctuation ;-)

File: src/translation/translation_es_ES.ts [hunk 3]
```diff
@@ -4703,12 +4704,13 @@ Hemos silenciado tu canal y activado &apos;Silenciarme Yo&apos;. Por favor resue
...
         <location filename="../sound/soundbase.cpp" line="171"/>
         <source>&lt;b&gt;%1 couldn&apos;t find a usable %2 audio device.&lt;/b&gt;&lt;br&gt;&lt;br&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;%1 no pudo encontrar un dispositivo de audio %2.&lt;/b&gt;&lt;br&gt;&lt;br&gt;</translation>
     </message>
     <message>
         <location filename="../sound/soundbase.cpp" line="182"/>
         <source>You may be able to fix errors in the driver settings. Do you want to open these settings now?</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Quizá puedas arreglar errores en la configuración del driver. ¿Quieres abrir la configuración ahora?</translatorcomment>
```
Comment (ann0see) [inline src/translation/translation_es_ES.ts:4712]: Comment not translation?

File: src/translation/translation_es_ES.ts (hunk 1 above)
Comment (ignotus666) [reply src/translation/translation_es_ES.ts]: Argh! It's happened to me before... Typed into the wrong field. Fixed.

Comment (pljones) [reply src/translation/translation_es_ES.ts]: > Typed into the wrong field.

Just for info, what tool are you using?

Comment (ignotus666) [reply src/translation/translation_es_ES.ts]: QtLinguist

**Review by ann0see — APPROVED** [review]
- [x] Translator listed in the `src/util.cpp`
- [x] Punctuation and spacing consistent (todo)
- [x] Signal words consistent ("ASIO", "Buffer") (todo)
- [ ] No untranslated strings (`grep unfinished -5 src/res/translation/translation_$TRANSLTION*.ts`)
- [x] Only a single `.ts` file checked in (+ Installer translations)

**Comment by ann0see** [comment]
Should still be squash merged ;-)

**Comment by pljones** [comment]
The problem with Github squash-commit is that it doesn't squash *merge* - it squashes to a single, non-merge, commit on master.

**Comment by ann0see** [comment]
Hmm. Yes. So you say it should be squashed locally and then merged?

**Comment by pljones** [comment]
It needs a rebase -i, the squash/fix whatever on the commits, then merge.  I was going to do it locally, yeah, the push directly.

**Pull request merged**